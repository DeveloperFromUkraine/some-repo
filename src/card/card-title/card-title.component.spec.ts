import { ComponentFixture, TestBed } from "@angular/core/testing";
import { CardTitleComponent } from "index";
import { ComponentTest } from "../../../test/test-bed/component";


describe('CardTitleComponent', () => {
    let fixture: ComponentFixture<CardTitleComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [CardTitleComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(CardTitleComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});