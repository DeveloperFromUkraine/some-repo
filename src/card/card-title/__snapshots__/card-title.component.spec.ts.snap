// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTitleComponent should match snapshot 1`] = `
<ign-card-title>
  <h1
    class="mat-card-title ign-card-title "
  >
    
    
    
  
  </h1>
  
</ign-card-title>
`;
